src/raven.js

```diff
@@ -435,21 +435,37 @@ Raven.prototype = {
       options
     );
 
-    if (this._globalOptions.stacktrace || (options && options.stacktrace)) {
-      var ex;
-      // Generate a "synthetic" stack trace from this point.
-      // NOTE: If you are a Sentry user, and you are seeing this stack frame, it is NOT indicative
-      //       of a bug with Raven.js. Sentry generates synthetic traces either by configuration,
-      //       or if it catches a thrown object without a "stack" property.
-      try {
-        throw new Error(msg);
-      } catch (ex1) {
-        ex = ex1;
-      }
+    var ex;
+    // Generate a "synthetic" stack trace from this point.
+    // NOTE: If you are a Sentry user, and you are seeing this stack frame, it is NOT indicative
+    //       of a bug with Raven.js. Sentry generates synthetic traces either by configuration,
+    //       or if it catches a thrown object without a "stack" property.
+    try {
+      throw new Error(msg);
+    } catch (ex1) {
+      ex = ex1;
+    }
 
-      // null exception name so `Error` isn't prefixed to msg
-      ex.name = null;
+    // null exception name so `Error` isn't prefixed to msg
+    ex.name = null;
+    var stack = TraceKit.computeStackTrace(ex);
 
+    var prvCall = stack.stack[1];
```

Comment:
I'd call this `initialCall` to give it more meaning. Also please add comment about why it's 2nd frame, not the top one – similar to here https://github.com/getsentry/raven-node/blob/master/lib/client.js#L337